charts/connectors/templates/tests/01-connectors-mm2-test.yaml

```diff
@@ -18,7 +18,7 @@ limitations under the License.
 apiVersion: v1
 kind: Pod
 metadata:
-  name: {{ include "connectors.fullname" . }}-mm2-test-{{ randNumeric 3 }}
```

Comment:
This random numbers is important as there are cases where helm install can cause problems with tests after helm upgrade. That's why I randomized names so that CR will print everything.

Reply:
With the `before-hook-creation` annotation, the pod with the same name will be deleted first.